best-practices/require-ro-rootfs/good-podcontrollers.yaml

```diff
@@ -41,4 +41,4 @@ spec:
           - name: busybox-again
             image: busybox:1.35
             securityContext:
-              readOnlyRootFilesystem: true
\ No newline at end of file
+              readOnlyRootFilesystem: true
```

Comment:
If we're not materially changing code, we should not be submitting these sort of changes to a public repo.  It just leads to more noise when we merge from public upstream. Can this change be reverted?